boards/shields/wiznet_wiz550io/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Grant Ramsay
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+# Allow access to private W5500 Ethernet header
+zephyr_library_include_directories(${ZEPHYR_BASE}/drivers/ethernet)
```

Comment:
looks like a hack, needs a better solution

Reply:
do you have an idea?

Reply:
@gramsay0 I proposed some ideas in other comments to help address this comment